docs/src/pages/demos/dividers/dividers.md

```diff
@@ -16,10 +16,21 @@ You can save rendering this DOM element by using the `divider` property on the `
 
```

Comment:
This describes using the `divider` prop of the `ListItem`, but the demo uses the `Divider` component.

Reply:
Good catch! This demo was already here so the problem is in master too but I don't mind fixing this here 👍